src/test/regress/sql/multi_complex_expressions.sql

```diff
@@ -158,3 +158,46 @@ SELECT count(*) FROM lineitem JOIN orders ON l_orderkey = o_orderkey
 -- Check that we make sure local joins are between columns only.
 
 SELECT count(*) FROM lineitem, orders WHERE l_orderkey + 1 = o_orderkey;
+
+-- Check that we can issue limit/offset queries
+
+-- SIMPLE LIMIT/OFFSET with ORDER BY
+SELECT o_orderkey FROM orders ORDER BY o_orderkey LIMIT 10 OFFSET 20;
+
+-- LIMIT/OFFSET with a Subquery
+SET citus.task_executor_type TO 'task-tracker';
+SELECT 
+	t.o_custkey,
+	COUNT(*) AS ss
+FROM 
+	(SELECT o_custkey, COUNT(*) AS order_count FROM orders GROUP BY o_custkey) t
```

Comment:
Can we use a more descriptive name for subquery instead of just "t"?
subtable, customer_count etc.